dydx/client.py

```diff
@@ -765,8 +765,87 @@ def get_perpetual_markets(
         '''
         Get all markets
 
-        :returns: { markets : [market: PerpetualMarket] }
+        :returns: { markets : [market: str]: PerpetualMarket } }
 
         :raises: DydxAPIError
         '''
         return self._get('/v1/perpetual-markets')
+
+    def get_funding_rates(
+        self,
+        markets=None,
+    ):
+        '''
+        Get the current and predicted funding rates.
+
+        IMPORTANT: The `current` value returned by this function is not active
+        until it has been mined on-chain, which may not happen for some period
+        of time after the start of the hour. To get the funding rate that is
+        currently active on-chain, use the get_perpetual_market() or
+        get_perpetual_markets() function.
+
+        The `current` rate is updated each hour, on the hour. The `predicted`
+        rate is updated each minute, on the minute, and may be null if no
```

Comment:
Do we want to remove `minute on the minute` if we intend to change it later?

Reply:
I don't think we intend to change it -- the premium is calculated at a random time, and then the predicted rate should change at the end of the interval (on the minute)